infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -103,11 +103,6 @@
 
     private final ServerContext serverContext;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "serverContext" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)